added AS13704

Canwail-Modular opened the pull request:
added ALFA As13704
Symbol is based on that of the LM13700
http://www.alfarzpp.lv/eng/sc/AS13704.pdf
![grafik](https://user-images.githubusercontent.com/50383630/67642453-370c7d80-f90c-11e9-8113-7b212690d428.png)

KLC Scripts tell me that 3.2 and 3.6 are violated, but that is the case for almost every device, so that seems to not be an issue with the symbol itself. 


---

All contributions to the kicad library must follow the [KiCad library convention](http://kicad-pcb.org/libraries/klc/)

Thanks for creating a pull request to contribute to the KiCad libraries! To speed up integration of your PR, please check the following items:

- [x] Provide a URL to a datasheet for the symbol(s) you are contributing
- [x] An example screenshot image is very helpful
- [x] Ensure that the associated footprints match the [official footprint library](https://github.com/kicad/kicad-footprints)
  - A new fitting footprint must be submitted if the library does not yet contain one.
- [ ] If there are matching footprint PRs, provide link(s) as appropriate
- [x] Check the output of the Travis automated check scripts - fix any errors as required
- [x] Give a reason behind any intentional library convention rule violation.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/KiCad/kicad-symbols?pullRequest=2269) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/KiCad/kicad-symbols?pullRequest=2269) before we can accept your contribution.<br/><hr/>**J. F** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/KiCad/kicad-symbols?pullRequest=2269) it.</sub>

cpresser commented:
Thanks for you contribution.
It looks pretty good. A few things should be improved though:
- [ ] Add the NC pins (7, 18). If they are present the footprint-filter based on pin-count will work properly. I suggest you add them to UnitA, according to KLC S4.6
- [ ] The Travis-CI datasheet-error should be fixed.

Canwail-Modular commented:
![grafik](https://user-images.githubusercontent.com/50383630/67966295-f0689d00-fc03-11e9-9d45-2025dfe113a3.png)

I added the NC pins as seen in your image above and moved the datasheet and info text down a notch.

cpresser commented:
Thanks!
